addons/website/static/tests/tours/specific_website_editor.js

```diff
@@ -2,29 +2,27 @@ odoo.define('website.tour.specific_website_editor', function (require) {
 'use strict';
 
 const tour = require('web_tour.tour');
+const wTourUtils = require("website.tour_utils");
 
-tour.register('generic_website_editor', {
+wTourUtils.registerWebsitePreviewTour("generic_website_editor", {
     test: true,
+    edition: true,
 }, [{
-    content: 'Click edit button',
-    trigger: '.o_edit_website_container > a',
-},
-{
     trigger: 'iframe body:not([data-hello="world"])',
-    extra_trigger: '#oe_snippets.o_loaded',
     content: 'Check that the editor DOM matches its website-generic features',
     run: function () {}, // Simple check
 }]);
 
+// Good practice would have been to use `wTourUtils.registerWebsitePreviewTour`
+// for this tour with `edition: true` and remove the first step to enter edit
+// mode. Unfortunately this breaks the page and therefore the test fails for
+// unknown reason.
```

Comment:
Maybe this is related to the bug of [task-3100228](https://www.odoo.com/web#id=3100228&cids=1&menu_id=4720&action=333&active_id=974&model=project.task&view_type=form)